js/hw_23.js

```diff
@@ -0,0 +1,282 @@
+alert(LEAP_YEAR);
+
+//  5
+const num5Digit = +prompt("Please, enter a five-digit number:");
+let a5 = num5Digit % 10;
+let z = num5Digit % 100;
+let a4 = (z - a5) / 10;
+let y = num5Digit % 1000;
+let a3 = (y - z) / 100;
+let x = num5Digit % 10000;
+let a2 = (x - y) / 1000;
+let a1 = (num5Digit - x) / 10000;
+if (num5Digit > 9999 && num5Digit < 100000 && a5 === a1 && a2 == a4) {
+  alert(`Your number ${num5Digit} is a palindrome.`);
+} else if (num5Digit < 10000 || num5Digit > 99999) {
+  alert(`That's not a five-digit number`);
+} else {
+  alert(
+    `Your number ${num5Digit} is not a palindrome: ${a5}${a4}${a3}${a2}${a1} `
+  );
+}
+
+//  6
+const moneyAmount = +prompt(
+  "This is a currency converter. Please enter an amount $:"
+);
+const currencyType = prompt("Please, ener a type of currency (UAH, EUR, ISK):");
+const CUR_RATE_UAH = 27.75;
+const CUR_RATE_EUR = 0.82;
+const CUR_RATE_ISK = 124;
+
+switch (currencyType) {
+  case "UAH":
+    alert(`You will get ${moneyAmount * CUR_RATE_UAH} Ukrainian Hryvnias`);
+    break;
+  case "EUR":
+    alert(`You will get ${moneyAmount * CUR_RATE_EUR} Euros`);
+    break;
+  case "ISK":
+    alert(`You will get ${moneyAmount * CUR_RATE_ISK} Icelandic Kronas`);
+    break;
+  default:
+    alert(`Please, enter the correct information`);
+}
+
+// 7
+const purchase = +prompt("Are you buying something? enter the price:");
+const SALE_3 = purchase * 0.03;
+const SALE_5 = purchase * 0.05;
+const SALE_7 = purchase * 0.07;
+if (purchase >= 200 && purchase < 300) {
+  alert(
+    `With an additional 3% discount it will cost ${(purchase - SALE_3).toFixed(
+      2
+    )}`
+  );
+} else if (purchase >= 300 && purchase < 500) {
+  alert(
+    `With an additional 5% discount it will cost ${(purchase - SALE_5).toFixed(
+      2
+    )}`
+  );
+} else if (purchase >= 500) {
+  alert(
+    `With an additional 7% discount it will cost ${(purchase - SALE_7).toFixed(
+      2
+    )}`
+  );
+} else {
+  alert(`The discount does not apply here.`);
+}
+
+// 8
+const PI = 3.14159;
+const circleCirc = +prompt("What is a circumference of a circle?");
+const squarePer = +prompt("What is a perimetr of a squre?");
+let circleDiameter = circleCirc / PI;
+let squareSide = squarePer / 4;
+let circleInside =
+  circleDiameter < squareSide
+    ? "Congrats! Your circle can fit into your square"
+    : "Sorry, your circle is too big for the square";
+alert(circleInside);
+
+// 9
+const question1 = +prompt(`Ready to answer some questions?
+(write the number of your answer)
+Question #1: Who is the author of the "Earthsea trilogy"?
+    1. J. R. R. Tolkien
+    2. C. S. Lewis
+    3. Ursula K. Le Guin `);
+const question2 =
+  +prompt(`Question #2: What is the name of Katara's brother in the cartoon "The Last Airbender"?
+    1. Sokka
+    2. Zuko
+    3. Aang `);
+const question3 = +prompt(`What year did Apollo 11 landed on the Moon?
+    1. 1961
+    2. 1969
+    3. 1990 `);
+
+var a = 0;
+var b = 0;
+var c = 0;
```

Comment:
```suggestion
let a = 0;
let b = 0;
let c = 0;
```
або просто 
```js
let score = 0;
```
а потім робити 
```js
score += 2;
```